index.rst

```diff
@@ -90,11 +90,13 @@ The types of lines are:
      ``[`` and ``]`` and even spaces and tabs are allowed).
    - Forward slashes (``/``) are used as path separators.
    - Backslashes (``\\``) are not allowed as path separators (even on Windows).
-- Key-Value Pair (or Pair): contains a key and a value, separated by an ``=``.
-   - Key: The part before the first ``=`` (trimmed of whitespace, but including
-     any whitespace in the middle).
-   - Value: The part after the first ``=`` (trimmed of whitespace, but including
-     any whitespace in the middle).
+- Key-Value Pair (or Pair): contains a key, an ``=``, and optionally a value.
+   - Key: The part before the first ``=`` on the line.
+   - Value: The part after the first ``=`` on the line, if any.
```

Comment:
```suggestion
   - Value: The part, if any, after the first ``=`` on the line.
```
otherwise someone could think the `=` might be optional even if this was required above.